tasks/install.packages-debian.yml

```diff
@@ -1,28 +1,43 @@
 ---
 - name: Check pdns_rec_repo_provider
-  fail: msg="pdns_rec_repo_provider is neither 'powerdns' nor 'os'"
+  fail:
+    msg: "pdns_rec_repo_provider is neither 'powerdns' nor 'os'"
   when: pdns_rec_repo_provider not in ["os", "powerdns"]
 
 # XXX: Is this handy to check ( I guess... no)
 - name: Check pdns_rec_repo_branch
-  fail: msg="pdns_rec_repo_branch isn't set to a known branch"
+  fail:
+    msg: "pdns_rec_repo_branch isn't set to a known branch"
   when: pdns_rec_repo_provider == "powerdns" and pdns_rec_repo_branch not in ["master", "40"]
 
 - name: Add the PowerDNS repository key
-  apt_key: url=http://repo.powerdns.com/CBC8B383-pub.asc id=D47975F8DAE32700A563E64FFF389421CBC8B383
+  apt_key:
+    url: http://repo.powerdns.com/CBC8B383-pub.asc
+    id: D47975F8DAE32700A563E64FFF389421CBC8B383
   when: pdns_rec_repo_provider == "powerdns" and pdns_rec_repo_branch == "master"
 
 - name: Add the PowerDNS repository key
-  apt_key: url=http://repo.powerdns.com/FD380FBB-pub.asc id=9FAAA5577E8FCF62093D036C1B0C6205FD380FBB
+  apt_key:
+    url: http://repo.powerdns.com/FD380FBB-pub.asc
+    id: 9FAAA5577E8FCF62093D036C1B0C6205FD380FBB
   when: pdns_rec_repo_provider == "powerdns" and pdns_rec_repo_branch in ["40"]
 
 - name: apt-pin the PowerDNS packages to the PowerDNS repo
-  copy: dest='/etc/apt/preferences.d/pdns' src=pdns.pin owner=root group=root mode=0644
+  copy:
+    dest: '/etc/apt/preferences.d/pdns'
+    src: pdns.pin
+    owner: root
+    group: root
+    mode: 0644
   when: pdns_rec_repo_provider == "powerdns"
 
 - name: Add the PowerDNS repository
-  apt_repository: repo="deb http://repo.powerdns.com/{{ansible_lsb.id | lower}} {{ansible_distribution_release|lower}}-rec-{{pdns_rec_repo_branch}} main"
+  apt_repository:
+    repo: "deb [arch=amd64] http://repo.powerdns.com/{{ ansible_lsb.id | lower }} {{ ansible_distribution_release | lower }}-rec-{{ pdns_rec_repo_branch }} main"
```

Comment:
This is because some Debian installations (and I believe Ubuntu by default) are multiarch. If you don't specify this in the repo line, apt will try to download all lists for all it's know architectures. And because we don't have i368 repos, this breaks.

However, we do also have armhf architecture repos for raspbian (but these are broken at the moment), so we'll need to fix that in a future PR